### PR TITLE
CLC-5075 - E-Grades Export 2: Add option to unmute assignments from error page

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.ets.berkeley.edu/jira/browse/CLC-5075

### CODE REVIEW

File: src/assets/templates/canvas_embedded/course_grade_export.html [hunk 1]
```diff
@@ -128,27 +128,46 @@ <h1 class="cc-page-course-grade-export-header">Required adjustments before downl
...
               E-Grade.csv exports cannot be properly calculated when assignments within the Gradebook are muted.<br>
               <a href="http://guides.instructure.com/m/4152/l/45130-how-do-i-mute-an-assignment-in-the-gradebook#UnmuteAssignment">How to mute assignments in Gradebook</a>
             </p>
 
+            <div class="cc-page-course-grading-export-unmute-action-container">
+              <div class="cc-page-course-grading-export-error-resolution-action-container">
+                <input id="cc-page-course-grading-export-enable-grading-scheme" type="checkbox" data-ng-model="unmuteAllAssignments">
+              </div>
+              <div class="cc-page-course-grading-export-error-resolution-action-container">
+                <label for="cc-page-course-grading-export-enable-grading-scheme" class="cc-page-course-grading-export-error-resolution-action-label">Unmute All</label><br>
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_grade_export.html]: @redconfetti does it still work without the extra `<br>` at the end?

File: src/assets/templates/canvas_embedded/course_grade_export.html [hunk 2]
```diff
@@ -128,27 +128,46 @@ <h1 class="cc-page-course-grade-export-header">Required adjustments before downl
...
+              </div>
+              <div class="cc-clearfix"></div>
```
Comment (christianv) [inline src/assets/templates/canvas_embedded/course_grade_export.html]: @redconfetti just a fyi, might be easier to add the `cc-clearfix-container` to the container containing the floats. (Definitely test before committing)

**Comment by christianv** [comment]
2 small comments, other parts of the front-end lgtm

**Comment by raydavis** [comment]
RSpec error in Travis, @redconfetti

**Pull request merged**